src/components/Todo.jsx

```diff
@@ -1,11 +1,10 @@
 import {AddTodo} from "./AddTodo.jsx"
-import {useState} from "react";
 import EditIcon from '@mui/icons-material/Edit'
 import DeleteIcon from '@mui/icons-material/Delete';
 import {Button, Typography, Box} from "@mui/material";
 import {useDispatch, useSelector} from "react-redux";
 import {delAllTasks,removeTask, toggleStatus} from "../redux/todo/actions/index.js";
-import {UpdateTaskForm} from "./common/UpdateTaskForm.jsx";
+import {useState} from "react";
```

Comment:
Re arrange the libraries order in this way.
/ External libraries
import { Button, Box, Typography } from "@mui/material";
import EditIcon from "@mui/icons-material/Edit";
import DeleteIcon from "@mui/icons-material/Delete";
import { useState } from "react";
import { useDispatch, useSelector } from "react-redux";

// Internal components
import { AddTodo } from "./AddTodo.jsx";
import { UpdateTaskForm } from "./common/UpdateTaskForm.jsx";

// Local state
import { delAllTasks, removeTask, toggleStatus } from "../redux/todo/actions/index.js";